Add custom rules + diff aware doc (#28053)
DataDog/documentation: content/en/security/code_security/troubleshooting/_index.md

If you are using Datadog's analyzer, [diff-aware scanning][6] is enabled by default. If you running the tool within your CI pipeline, make sure that `datadog-ci` runs **at the root** of the repository being analyzed.

### Diff-aware is not working

If diff-aware is not working with the Static Analyzer, ensure that:

>  1. Your default branch is specific for your repository.

Reviewer: ```suggestion
 1. The default branch is specific to your repository.
```

>  2. One revision with the same configuration (for example, same rulesets, same arguments, only/ignore flags, etc.) has been pushed on the repository default branch.

Reviewer: ```suggestion
 2. At least one revision with the same configuration (for example, same rulesets, same arguments, or only/ignore flags) has been pushed to the repository's default branch.
```

> If diff-aware is not working with the static analyzer, make sure that
>  1. your default branch is specific for your repository
>  2. one revision with the same configuration (e.g. same rulesets, same arguments, only/ignore flags, etc.) has been pushed on the repository default branch.
>  3. The current user can read the repository metadata (use the following command to fix this: `git config --global --add safe.directory <repo-path>`)

Reviewer 2: ```suggestion
If diff-aware is not working with the Static Analyzer, ensure that:
 1. Your default branch is specific for your repository.
 2. One revision with the same configuration (for example, same rulesets, same arguments, only/ignore flags, etc.) has been pushed on the repository default branch.
 3. The current user can read the repository metadata. Use the following command to fix this: `git config --global --add safe.directory <repo-path>`.
```

>  3. The current user can read the repository metadata. Use the following command to fix this: `git config --global --add safe.directory <repo-path>`.

Reviewer: ```suggestion
 3. The current user can read the repository metadata. If they do not have the correct permissions, run this command: `git config --global --add safe.directory <repo-path>`.
```

### Disabling runtime SCA on your services

SCA can be enabled on your running services using one of the following two methods:

> - The Datadog UI.

Reviewer: ```suggestion
- From the Datadog UI.
```

Reviewer: Perhaps adding a link or navigation steps here would be helpful.
